.github/workflows/main-build.yml

```diff
@@ -461,6 +461,7 @@ jobs:
     uses: ./.github/workflows/appsignals-e2e-eks-test.yml
     secrets: inherit
     with:
+      aws-region: us-west-2
       test-cluster-name: "e2e-adot-test"
```

Comment:
The cluster `e2e-adot-test` does not exist in us-west-2. The region needs to be `us-east-1` and there needs to be a concurrency check to ensure that when this action is run the other workflows that use the same cluster do not run at the same time.

Reply:
I believe this PR is to explicitly address the workflow failure that is occurring in the main build workflow